### PR TITLE
open-cluster-management/backlog#18197--change to version

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Review by sahare — CHANGES_REQUESTED** [review]
Please update the OADP link to: https://github.com/openshift/oadp-operator/blob/oadp-0.4.2/docs/install_olm.md

File: clusters/backup_and_restore.adoc [hunk 1]
```diff
@@ -6,7 +6,11 @@ The cluster backup and restore operator runs on the hub cluster, and depends on
 [#prerequisites-backup-restore]
 == Prerequisites
 
-You must install the OADP Operator. See link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[Install OADP Operator using OperatorHub]. Be sure to complete the steps to link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-credentials-secret[create the secret] for the cloud storage, where the backups are saved. Then use the created secret when you create the https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-the-velero-custom-resource[Velero resource].
+You must install the 0.4.2 version of the OADP Operator from the _beta_ channel. See link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[Install OADP Operator using OperatorHub].
```
Comment (birsanv) [inline clusters/backup_and_restore.adoc]: The doc link should point to the 0.4.2 branch, not main

https://github.com/openshift/oadp-operator/blob/oadp-0.4.2/docs/install_olm.md

This :
link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[Install OADP Operator using OperatorHub].

should be 

link:https://github.com/openshift/oadp-operator/blob/oadp-0.4.2/docs/install_olm.md[Install OADP Operator using OperatorHub].

File: clusters/backup_and_restore.adoc [hunk 2]
```diff
@@ -6,7 +6,11 @@ The cluster backup and restore operator runs on the hub cluster, and depends on
 [#prerequisites-backup-restore]
 == Prerequisites
 
-You must install the OADP Operator. See link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[Install OADP Operator using OperatorHub]. Be sure to complete the steps to link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-credentials-secret[create the secret] for the cloud storage, where the backups are saved. Then use the created secret when you create the https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-the-velero-custom-resource[Velero resource].
+You must install the 0.4.2 version of the OADP Operator from the _beta_ channel. See link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[Install OADP Operator using OperatorHub]. 
+
+Be sure to complete the steps to link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-credentials-secret[create the secret] for the cloud storage, where the backups are saved. 
+
+Use the created secret when you create the https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-the-velero-custom-resource[Velero resource].
```
Comment (birsanv) [inline clusters/backup_and_restore.adoc]: Link should be 

https://github.com/openshift/oadp-operator/blob/oadp-0.4.2/docs/install_olm.md#create-the-velero-custom-resource

File: clusters/backup_and_restore.adoc [hunk 3]
```diff
@@ -6,7 +6,11 @@ The cluster backup and restore operator runs on the hub cluster, and depends on
 [#prerequisites-backup-restore]
 == Prerequisites
 
-You must install the OADP Operator. See link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[Install OADP Operator using OperatorHub]. Be sure to complete the steps to link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-credentials-secret[create the secret] for the cloud storage, where the backups are saved. Then use the created secret when you create the https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-the-velero-custom-resource[Velero resource].
+You must install the 0.4.2 version of the OADP Operator from the _beta_ channel. See link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md[Install OADP Operator using OperatorHub]. 
+
+Be sure to complete the steps to link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-credentials-secret[create the secret] for the cloud storage, where the backups are saved.
```
Comment (birsanv) [inline clusters/backup_and_restore.adoc]: Link should be https://github.com/openshift/oadp-operator/blob/oadp-0.4.2/docs/install_olm.md#create-credentials-secret

**Review by birsanv — COMMENTED** [review]
we need to update the link to point to the 0.4.2 branch

**Review by sahare — APPROVED** [review]
Looks good, Thanks!

**Review by birsanv — APPROVED** [review]
lgtm

File: clusters/backup_and_restore.adoc (hunk 1 above)
Comment (swopebe) [reply clusters/backup_and_restore.adoc]: changed all links

**Pull request merged**